rio-backend/src/config/colors/mod.rs

```diff
@@ -738,4 +733,31 @@ mod tests {
             }
         );
     }
+
+    #[test]
```

Comment:
That's great! thank you for the PR @bio , could add another test using `#06a49b99`?

Reply:
@raphamorim sure, I added an additional test case.

Reply:
Thank you!